.github/workflows/blui-ci.yml

```diff
@@ -0,0 +1,114 @@
+name: Build
+
+on:
+  push:
+    branches: ['dev', 'master']
+  pull_request:
+    branches: ['dev', 'master']
+  pull_request_target:
+    types:
+      - opened
+    branches:
+      - '*/*'
+
+permissions:
+  pull-requests: write
+  contents: read
+
+jobs:
+  prettier_lint:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [20.x]
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: 'yarn'
+      - run: yarn
+      # - run: yarn docs-prettier
+      #   working-directory: docs
+    #   - run: yarn docs-lint
+    #     working-directory: docs
+
+  build_rn_docs:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [20.x]
+    env:
+      base-href: ${{ github.ref == 'refs/heads/master' && '/react-native/' || '/react-native-dev/' }}
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: 'yarn'
+      - run: yarn --immutable
+      - run: yarn docs-build --base= ${{env.base-href}} && touch ./dist/docs/.nojekyll
```

Comment:
is this space after base= going to cause a problem?

Reply:
it shouldn't, this is what vite build recommends. Here is what the index looks like in the build.

<img width="700" alt="image" src="https://github.com/etn-ccis/blui-react-native/assets/119693939/646b0efa-784b-4e7e-94bb-571aa6c56c44">